Update setting_primary_tags_to_scope.md (#11019)
DataDog/documentation: content/en/tracing/guide/setting_primary_tags_to_scope.md

#### Agent environment

The `env` tag can be set in your Agent configuration.

> **We recommend never setting different `env` tags on the Tracer and Agent. This may cause duplicate tagging on [trace metrics][6].**

Author: ```suggestion
**Do not set different `env` tags on the Tracer and Agent. This may cause duplicate tagging on [trace metrics][6].**
```
